src/app/nation/nation.component.scss

```diff
@@ -0,0 +1,6 @@
+@import '../variables.scss';
+
+.form-control{
```

Comment:
form control is the bootstrap class. so why is there style for this specific class over here?

Reply:
to give style for input field